### PR TITLE
Destroy comment

### PR DESCRIPTION
**Pull request by Hanzolskl**
User which left comment should have ability to delete coment
https://github.com/elabs/pundit

### CODE REVIEW

File: app/views/posts/show.html.haml [hunk 1]
```diff
@@ -12,6 +12,7 @@
   - @post.comments.each do |comment|
     %li.list-group-item
       = comment.body
+      = link_to 'Destroy',[comment.post ,comment], method: :delete, data: { confirm: 'Are you sure?' }
```
Comment (Magons) [inline app/views/posts/show.html.haml]: use @post instead comment.psot
check you spaces

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -32,8 +32,13 @@ def update
   end
 
   def destroy
-    @comment.destroy
-    respond_with(@comment)
+    if (current_user.id == @comment.user_id)
```
Comment (Magons) [inline app/controllers/comments_controller.rb]: don't do this. For this you need use gem

Comment (Hanzolskl) [inline app/controllers/comments_controller.rb]: devise?

Comment (Magons) [inline app/controllers/comments_controller.rb]: pundit

Comment (Hanzolskl) [inline app/controllers/comments_controller.rb]: i don't have this gem in the my project

**Pull request closed without merging**